### PR TITLE
partial

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
- #120 add partial change of content
- fix partial not showing in part and expandable
- #122 toggle unselect

### CODE REVIEW

**Review by FabienLelaquais — APPROVED** [review]
I get it!
Nicely done

File: taipy/gui/gui_actions.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 from .gui import Gui
 from .state import State
+from .partial import Partial
```
Comment (dinhlongviolin1) [inline taipy/gui/gui_actions.py]: This import should be removed I guess

**Review by dinhlongviolin1 — APPROVED** [review]
Looks good to me! Simple yet effective solution for multi user

File: taipy/gui/gui_actions.py (hunk 1 above)
Comment (FredLL-Avaiga) [reply taipy/gui/gui_actions.py]: Good catch, tx

**Pull request merged**